pegasus/sites.v3/code.org/views/donor_slider.haml

```diff
@@ -9,17 +9,10 @@
 
 #donorslider
   .container_responsive
-    #donorheader{style: "position:relative; height:40px"}
+    .donor-header
       %h1
         = I18n.t(:homepage_header_donors)
 
-      #donorlinks.desktop-feature
-        %a{href: "/about/donors"}
-          = I18n.t(:homepage_donors_seall)
```

Comment:
I see that the mock removed it, but are the product/marketing/development teams okay with removing this link?

Reply:
I ran this final implementation by Whitney. It also looks like she confirmed this in the duplicate PR [here](https://codedotorg.atlassian.net/browse/PP-135?focusedCommentId=21461).

Reply:
Sounds good.